run_node.sh

```diff
@@ -216,6 +216,37 @@ else
     echo ""
     echo "Specified account do not have any session keys set."
     echo "You should generate and set the keys to be able to validate."
+
+    read -p "Do you want to generate new session keys? [y/N] " -n 1 -r
+    if [[ ${REPLY} =~ ^[Yy]$ ]]
+    then
+        echo ""
+        echo "Generating new session keys..."
+        NEW_KEYS_JSON=$(curl -H "Content-Type: application/json" -d '{"id":1,
+            "jsonrpc":"2.0", "method": "author_rotateKeys"}' http://127.0.0.1:"${RPC_PORT}")
+
+        NEW_KEYS=$(echo ${NEW_KEYS_JSON} | docker run -i "${JQ_IMAGE}" '.result' | tr -d '"')
+        echo "New session keys: ${NEW_KEYS}"
```

Comment:
Output of the script is pretty hard to read because of outputs from commands. Maybe it would be nice to print newlines in some important places like before this message and after this message. 

Also maybe additional newline before:
* 224: echo "Generating new session keys..."
* 138: echo 'Performing session key checks...'
* 108: echo "Running the node..."
* 101: echo "Pulling docker image..."

I know these last ones are not in scope of this PR, but it would make the output nicer.